src/translations/en_US.js

```diff
@@ -25,12 +26,14 @@ module.exports = {
   paypalTokenizationRequestActiveError: 'PayPal payment authorization is already in progress.',
   unsupportedCardTypeError: 'This card type is not supported. Please try another card.',
   // Card form
+  cardholderNameLabel: 'Cardholder Name',
   cardNumberLabel: 'Card Number',
   cvvLabel: 'CVV',
   cvvThreeDigitLabelSubheading: '(3 digits)',
   cvvFourDigitLabelSubheading: '(4 digits)',
   expirationDateLabel: 'Expiration Date',
   expirationDateLabelSubheading: '(MM/YY)',
+  cardholderNamePlaceholder: 'The Name on your Card',
```

Comment:
We should run this by the designers before we get translations, maybe we don't have a placeholder here?